code/modules/psionics/faculties/coercion.dm

```diff
@@ -300,7 +300,7 @@
 			if(!T || L == user || L.stat == DEAD || L.invisibility == INVISIBILITY_LEVEL_TWO)
 				continue
 			if(L.isSynthetic() || L.is_diona() || isvaurca(L))
-				var/obj/item/organ/internal/augment/psiaug = locate() in L
+				psiaug = has_psi_aug()
```

Comment:
You... forgot... to specify what actually the proc should be called on.

Reply:
also it returns boolean